recipes/simple-build-dgcode/build.sh

```diff
@@ -0,0 +1,13 @@
+set -eux
+#To follow conda-forge's best practices, we got the sources via a tarball, but
+#simple-build-dgcode uses setuptools-git-versioning which gets the version from
+#the git tag. So we initialise a git repo and add the version tag.
+git init
```

Comment:
If the project uses a versioning tool which doesn't work with sdists I'd be inclined to just use a git checkout rather than this hack.

@conda-forge/help-python may not agree with that? I'm not sure how religiously that "requirement" is supposed to be taken.

Reply:
This tool probably generates an `sdist` where the tag is baked in. You should be using the sdist instead of the git tarball as the source to build from.

Reply:
PyPI is the canonical source for Python packages, whenever possible. PyPI being down is as unlikely as catastrophical; we'd have bigger problems if that happens :)

Yes, the bots understand PyPI URL schemes. Follow the example provided for context.

Reply:
Ok, I have removed the "git init" hack and now use the pypi sdist package as the source :-)